lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -12,59 +12,79 @@
   }
 }
 
+%dot {
+  &:before {
+    content: "•";
+    color: $gray;
+    position: absolute;
+    left: .6rem;
+    opacity: 0;
+  }
+  &:hover, &:focus {
+    background-color: $off-white;
+    color: $darker-gray;
+    text-decoration: none;
+    &:before {
+      opacity: 1;
+    }
+  }
+  &.is-active {
+    font-weight: 600;
+    &:before {
+      opacity: 1;
+      color: $blue;
+    }
+  }
+}
+
   @mixin dropdown-link() {
-    position: relative;
-    display: block;
     @include box-sizing(border-box);
-    padding: $baseline/3;
     @include font-size(-2);
+    position: relative;
+    padding: $baseline/2.5 $baseline/1;
+    display: block;
     color: $darker-gray;
-    background-color: $white;
-    border-bottom: none;
-    text-align: left;
-    border-left: none;
-    border-right: none;
-    border-top: 1px solid $lightest-gray;
-    white-space: nowrap;
+    font-weight: inherit;
     cursor: pointer;
-    width: 100%;
-    &:hover {
-      background-color: $off-white;
-      text-decoration: none;
-      color: $darker-gray;
-    }
-    &.is-active, &:focus {
-      text-indent: -3px;
-      border-left: 3px solid $blue;
-    }
+    background-color: transparent;
+    overflow: visible;
+    @extend %dot;
+
     @if ($include-right-to-left) {
       html[dir="rtl"] & {
         text-align: right;
       }
-      html[dir="rtl"] &.is-active, html[dir="rtl"] &:focus {
-        border-left: none;
-        border-right: 3px solid $blue;
+      html[dir="rtl"] &:before {
+        left: unset;
+        right: .6rem;
       }
     }
   }
 
   @mixin dropdown-title() {
-    @include dropdown-link();
-    background-color: $off-white;
+    @include font-size(-2);
+    padding: $baseline/2.5 $baseline/1;
+    display: block;
+    font-weight: 600;
     cursor: auto;
+    border-bottom: 1px solid $lightest-gray;
+    margin: $baseline/3 $baseline/1.5 -1px $baseline/1.5;
+    padding-left: 0;
+    @if ($include-right-to-left) {
+      html[dir="rtl"] & {
+        padding-right: 0;
+      }
+    }
   }
 
   @mixin dropdown-btn {
-    cursor: pointer;
-    position: relative;
   }
```

Comment:
Is this used anymore? If not, let's remove from here and the include at the end of the file.